contract/astar/AstarBaseV3.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.7;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "./DappsStakingDummy.sol";
+import "./SR25519Dummy.sol";
+import "./ECDSADummy.sol";
+
+/// @author The Astar Network Team
+/// @title Astarbase. A voluntary mapping of accounts ss58 <> H160
+contract AstarBaseV3 is Initializable, OwnableUpgradeable {
```

Comment:
What is the puropse of V3 file?

Reply:
There is a bugfix in register method

instead of concatinating bytes32 public key in full message we need to concatinate bytes. 

Please deploy v3 as it is necessary for ecdsa to work.